### PR TITLE
Better docs for fromnow abbreviations

### PR DESCRIPTION
**Pull request by djmitche**
(incorporating suggestions in #483 from @gregsdennis)

### CODE REVIEW

**Review by gregsdennis — APPROVED** [review]

File: docs/src/operators.md [hunk 1]
```diff
@@ -122,17 +122,15 @@ template: {$fromNow: '1 hour', from: '2017-01-19T16:27:20.974Z'}
...
+The available units, including useful shorthands, are:
 
 ```none
 years,    year,   yr,   y
 months,   month,  mo
 weeks,    week,   wk,   w
 days,     day,          d
 hours,    hour,   hr,   h
-minutes,  minute, min
+minutes,  minute, min   m
```
Comment (gregsdennis) [inline docs/src/operators.md:133]: Nitpick:  comma after `min`?

**Comment by djmitche** [comment]
Still to do:

The Python and Rust `fromNow` implementations require that the offsets be in order from largest to smallest - `1y 2w` is OK, but `2w 1y` is not. 

Also, negative dates have a `-` prefix. Go and JS allow a `+` prefix as well (which does nothing). 

Also, double-check that none of the implementations accidentally include additional abbreviations that the others do not.

**Comment by djmitche** [comment]
> The Python and Rust `fromNow` implementations require that the offsets be in order from largest to smallest - `1y 2w` is OK, but `2w 1y` is not.

All implementations agree on this.
 
> Also, negative dates have a `-` prefix. Go and JS allow a `+` prefix as well (which does nothing).

Rust and Python do as well.

> Also, double-check that none of the implementations accidentally include additional abbreviations that the others do not.

Done.

**Review by gregsdennis — COMMENTED** [review]
New notes look good. Thanks.

**Review by petemoore — APPROVED** [review]
Looks great, many thanks Dustin!

**Comment by petemoore** [comment]
@djmitche Are you happy to look at the merge conflict?

**Pull request merged**